addons/base_vat/models/res_partner.py

```diff
@@ -224,6 +224,20 @@ def check_vat_al(self, vat):
             return True
         return False
 
+    __check_vat_hu_re = re.compile(r'^\d{8}-\d-\d{2}$')
```

Comment:
The single digit number can be only `1` or `2`
I believe that the last 2 digits can only be a small subset too, including only 1, 2 and 3 as digits.
https://en.wikipedia.org/wiki/NUTS_statistical_regions_of_Hungary

Reply:
For the last two digit it can be other digit i think 
![image](https://github.com/odoo/odoo/assets/109144554/eb9e13eb-bded-464b-a540-eda63d5e22a0)